CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+lvm-localpv-1.6.2 / 2024-09-19
+========================
+<b>NOTE</b>: This was only a chart release that addressed a bug on prior chart. 
+* fix(chart): revert env OPENEBS_NAMESPACE to LVM_NAMESPACE for v1.6.x ([#333](https://github.com/openebs/lvm-localpv/pull/333),[@niladrih](https://github.com/niladrih))
```

Comment:
This is not a version for the project. This is a convenient tool we use to host the helm chart on the GitHub release page. Should this be a part of v1.6.1, because that's the code version?

I am not sure what the better practice would be here. WDYT @tiagolobocastro @Abhinandan-Purkait?

Reply:
Yeah, I was also unsure what to do here? Should I remove it?

Reply:
Keeping it for now.